pages/home.vue

```diff
@@ -58,7 +58,7 @@ export default {
         labelKey: 'nav.header.setLoginPage',
         action:   SET_LOGIN_ACTION
       },
-      { seperator: true },
+      { separator: true },
```

Comment:
??

Reply:
Marking this one to come back to for closer review.